doc/user-guide/template.md

```diff
@@ -225,7 +225,7 @@ There is a second method to parametrize variables via `:variable`. This method h
 To enable it:
 
 ```{code-cell} ipython3
-%config SqlMagic.named_parameters = True
+%config SqlMagic.named_parameters = "true"
```

Comment:
Is it okay if we change this to:
`0`: False
`1`: True
`2`: Disabled

Users might confuse the string version with boolean.

@edublancas @bryannho

Reply:
[Implemented this change here](https://github.com/ploomber/jupysql/pull/980/files#diff-527caf90dd08297f5253e826bd98fea0b6f4fbd44fe9f20472976f77868dac06R159-R166) and documented it [here](https://github.com/ploomber/jupysql/pull/980/files#diff-7c559917b46c06e53b27fa5152bf29cb1db41f04c53f857ef5331f7a003204ffR262-R267)